gating/update_dependencies/release-update.py

```diff
@@ -52,11 +52,7 @@
 # standard.
 
 release_naming_standard = re.compile(
-    "^r[0-9]+.[0-9]+.[0-9]+(-(alpha|beta).[0-9]+)?$")
-
-assert release_naming_standard.match(rpc_release), (
```

Comment:
Why is this assertion being removed? It should stay to verify the version matches the standard.